defaults/main.yml

```diff
@@ -0,0 +1,13 @@
+---
+nim_waku_sonda_cont_name: 'sonda'
+nim_waku_sonda_cont_vol: '/docker/{{ nim_waku_sonda_cont_name }}'
+nim_waku_sonda_metrics_port: 8004
+nim_waku_sonda_service_name: 'sonda'
+nim_waku_sonda_github_url_path: 'https://raw.githubusercontent.com/waku-org/nwaku/master/apps/sonda'
```

Comment:
Don't just use `master` or it will break in the future. Add a commit variable and include a hash for the file.